packages/beeq/src/components/select/_storybook/bq-select.stories.tsx

```diff
@@ -257,6 +268,16 @@ export const Multiple: Story = {
   },
 };
 
+export const MultipleCustomRender: Story = {
+  render: Template,
+  args: {
+    'keep-open-on-select': true,
+    multiple: true,
+    customTags: true,
+    value: ['pizza', 'hamburger', 'cookie'],
+  },
+};
```

Comment:
Can a functional example be provided that syncs with the select/unselect from the list?

Reply:
Done.